client/jetpack-connect/site-topic.js

```diff
@@ -26,7 +26,7 @@ class JetpackSiteTopic extends Component {
 
 		this.props.saveSiteVertical( siteId, siteVertical );
 
-		page( `/jetpack/connect/plans/${ siteSlug }` );
+		page( `/jetpack/connect/user-type/${ siteSlug }` );
```

Comment:
Hm, this new step requires Jetpack 7.2, so we should redirect to `/user-type` only for Jetpack >= 7.2, and keep the `/plans` redirect for older Jetpacks.

Reply:
good call